Client/Frontend/Browser/TabDisplayManager.swift

```diff
@@ -390,11 +392,17 @@ extension TabDisplayManager: TabManagerDelegate {
         }
     }
 
-    func tabManager(_ tabManager: TabManager, willRemoveTab tab: Tab) {
-        isDragging = false
-    }
+    func tabManager(_ tabManager: TabManager, willAddTab tab: Tab) {}
```

Comment:
willAddTab and willRemoveTab were only being used by TabDisplayManager. We can remove them completely. Also make sure to remove any tests that use them

Reply:
Less code is good! will remove...